src/event-manager.js

```diff
@@ -111,17 +111,20 @@ export default class EventManager {
 
   // Tear down internal event management implementations.
   destroy() {
-    this.element.removeEventListener('contextmenu', preventDefault);
-
-    this.wheelInput.destroy();
-    this.moveInput.destroy();
-    this.keyInput.destroy();
-    this.manager.destroy();
-
-    this.wheelInput = null;
-    this.moveInput = null;
-    this.keyInput = null;
-    this.manager = null;
+    if (this.element) {
+      this.element.removeEventListener('contextmenu', preventDefault);
```

Comment:
Can you add a comment that mentions that `wheelInput` etc are created in `setElement` and therefore cannot exist if there is no element? JIC that's not obvious to anyone in the future.